dts/bindings/audio/nxp,dmic.yaml

```diff
@@ -32,12 +32,12 @@ child-binding:
   description: |
     NXP DMIC channel. Can be used to configure filtering and gain attributes
     of each channel
-  include: base.yaml
```

Comment:
This need not be deleted

Reply:
Why? Again, this is for the **child-binding** so having this include is actually causing the **required** `compatible` property to be inherited which we don't want as dts files then complain about the missing `compatible`